src/engine/ruleset/integrations/windows/powershell/decoders/windows-powershell-operational.yml

```diff
@@ -0,0 +1,79 @@
+name: decoder/windows-powershell-operational/0
+
+sources:
+  - decoder/windows-event-decoder/0
+
+metadata:
+  description: Decoder for Windows Powershell Operational events
+  module: Windows Powershell/Operational
+  title: Decoder for Windows Powershell Operational logs
+  compatibility: >
+    This decoder has been tested on Wazuh version 4.3
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/23
+  references:
+    - https://learn.microsoft.com/en-us/powershell/scripting/windows-powershell/wmf/whats-new/script-logging?view=powershell-7.3
+
+check: ~windows.Event.System.Channel.#text==Microsoft-Windows-PowerShell/Operational
+
+normalize:
+ - map:
+   - event.module: windows
+   - event.dataset: powershell
+   - event.category: [process]
+   - event.type: [info]
+   - event.sequence: $~windows.Event.EventData.SequenceNumber
+
```

Comment:
In order to improve traceability, it may be useful to add to the `wazuh.decoders` field the name of each decoder that processes the event:
`- wazuh.decoders: +array_append/windows-powershell-operational`
Apply the same change for all the decoders of the integration.

Reply:
Done